performance-storage-service/Jenkinsfile

```diff
@@ -21,44 +25,42 @@ node {
       }
 
       stage('Test'){
-        sh 'virtualenv env -p python3.7'
-        sh '. env/bin/activate'
-        sh 'pip install -r requirements.txt'
-        sh 'python3 manage.py test'
-        sh 'env/bin/deactivate'
+        agent {
+            docker { image 'maven:3-alpine' }
```

Comment:
hmmm... i thought maven is for java projects and this is a python one?

Reply:
whoops thats a copy paste error. I'm looking into if there is a good way to run the tests in a docker image